### DISCUSSION TITLE
Rancher Desktop invoke rosetta on first run

### DISCUSSION THREAD

**Discussion by theLastOfCats** [rancher-sandbox/rancher-desktop#1409] [Q&A] [answered]
What for?
Can we make it fully native for M1?

**Comment by jandubois** [comment, marked as answer]
> What for?

The GUI code is currently using Electron. The only other part is the `kuberlr` and `kubectl` binaries, as there are no official releases of `kubectl` older than 1.21.0.

> Can we make it fully native for M1?

That is the plan, but it is not clear yet how to distribute `kubectl` versions compatible with older k8s releases.

This doesn't seem urgent, as all the performance-critical parts are fully native already, and the GUI just has the Rosetta delay on the first start.
